packages/plugins/flow/flow/src/visitor.ts

```diff
@@ -100,7 +100,17 @@ export class FlowVisitor extends BaseTypesVisitor<FlowPluginConfig, FlowPluginPa
       return allFields.join('\n');
     }
 
-    return `...{\n${allFields.map(s => indent(s)).join('\n')}\n  }`;
+    return (
+      new DeclarationBlock(this._declarationBlockConfig)
+        .withContent('...')
+        .withBlock(allFields.join('\n'))
+        .ignoreBlockSemicolon(true)
+        .string.slice(0, -1) // Removes trailing newline from stringified block.
```

Comment:
It would be great to have the trailing newline configurable as well, the same way block semicolon was added by me now.